src/lib.rs

```diff
@@ -155,26 +188,23 @@
 //!                     "Received an MMIO Write Request to the address {:#x}.",
 //!                     addr,
 //!                 );
-//!             }
-//!             VcpuExit::Hlt => {
 //!                 // The code snippet dirties 1 page when it is loaded in memory
 //!                 let dirty_pages_bitmap = vm.get_dirty_log(slot, mem_size).unwrap();
 //!                 let dirty_pages = dirty_pages_bitmap
 //!                     .into_iter()
 //!                     .map(|page| page.count_ones())
 //!                     .fold(0, |dirty_page_count, i| dirty_page_count + i);
 //!                 assert_eq!(dirty_pages, 1);
+//!                 #[cfg(target_arch = "aarch64")]
+//!                 break;
```

Comment:
On aarch64 you don't have a halt instruction :(

Reply:
I think we should still add a comment here to say that this break it's only there for the example (so it doesn't loop forever), because in an actual VMM loop you will not want to exit on MMIO writes.

Reply:
Done, take a look.